content/en-us/build/general/intro.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "Introduction"
+weight: 1001
+draft: false
+menu:
+  build:
+    parent: "phat-general"
+---
+
+## What is Phat Contract
+
+Phat Contract is the innovative programming model enabling [*Off-chain Computation*](https://medium.com/phala-network/fat-contract-introduce-off-chain-computation-to-smart-contract-dfc5839d5fb8). It’s also known as Fat Contract as a practice of the "Fat Protocol & Thin Application" concept, and for its rich functionalities compared with existing smart contract. Phat Contract uses Rust-based [Ink! language](https://paritytech.github.io/ink/).
+
+Although with the name of "contract", Phat Contract is more like the Web3 version of [Amazon Lambda](https://aws.amazon.com/lambda/) backed by the decentralized Phala computing cloud, and meant to support complex computation with low latency and cost.
+
+Join our [Discord server](https://discord.com/invite/phala) and learn more in the `#dev` channel.
+
+## Why using Phat Contract
+
+### Compared with Smart Contract
```

Comment:
Should address Phat Contract has no mean to replace or compete with smart contract. Instead, it's complementary to smart contracts. The current logic address Phat Contract is better than smart contract (note that you mentioned "fix")

Reply:
updated